e2e/app/component/navigation.ts

```diff
@@ -35,7 +35,7 @@ export default class Navigation {
     const findMenuItem = async (): Promise<ElementHandle | null> => {
       await Navigation.openNavMenu(page);
       const angleIconXpath = buildXPath({type: ElementType.Icon, iconShape: 'angle'});
-      await page.waitForXPath(angleIconXpath, {timeout: 2000});
+      await page.waitForXPath(angleIconXpath, {timeout: 2000, visible: true});
```

Comment:
nit: we can have an input as a common variable and use it just so in case it changes in future you have to change in one place